python/test/data_extraction/test_export_data.py

```diff
@@ -47,8 +47,12 @@ def test_export_content(self):
         content_store_has_item(content_second['base_path'], json=content_second)
         with unittest.mock.patch('gzip.open', return_value=output):
             export_data.export_content()
+
+            def sort_export(lst):
+                sorted(lst, key=lambda x: x['base_path'])
+
             expected = [content_first, content_second]
-            self.assertEqual(expected, json.loads(output.buffer))
+            assert sort_export(expected) == sort_export(json.loads(output.buffer))
```

Comment:
I think assertCountEqual is the correct method
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual

Reply:
That would check the count, but it wouldn't compare what the things in the two lists are. I don't think we want to loose that part of the expectation?